### DISCUSSION TITLE
Rewrite in JavaScript to make an extension

### DISCUSSION THREAD

**Discussion by TechStudent10** [ThioJoe/YT-Spammer-Purge#88] [Ideas]
As @JamesPerisher said:
> I had a though that I think would be a good direction to take the project. If the application where to be written/ported to javascript then it can be a quite easily made into a browser extension that would work on all platforms allowing total modification of the YouTube comment section allowing the delete all messages by userxyz to be a direct button in the youtube.com comment section. By doing this the entire concept of a gui is nolonger needed as an item can simply be added to an existing dropdown like so: ![image](https://user-images.githubusercontent.com/37967469/142713969-833d7c58-69f8-48e0-af1f-1ae578784d0e.png)

_(Originally posted in #84)_

I honestly also think this is a good idea.

**Comment by eilon40** [comment]
https://github.com/fknives/Youtube-Spam-Purge-Almost-Chrome-Extension

**Reply by TechStudent10** [reply]
Great find! Can't believe someone actually went ahead and did this!
